packages/launchpad/src/global/GlobalPageHeader.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import { defaultMessages } from '@cy/i18n'
+import GlobalPageHeader from './GlobalPageHeader.vue'
+import { ref } from 'vue'
+
+const searchSelector = `input[placeholder="${defaultMessages.globalPage.searchPlaceholder}"]`
+
+type FileWebKitDirectory = File & { path: string }
+
+export default function attachFileToInputElement (element: HTMLInputElement) {
+  const file = new File([new Blob()], 'cypress.json') as FileWebKitDirectory
+
+  file.path = 'absolute/path/to/cypress.json'
+
+  const dataTransfer = new window.DataTransfer()
+
+  dataTransfer.items.add(file)
+
+  element.files = dataTransfer.files
+}
```

Comment:
When you build the tests for the parent, you'll probably want to put this into a custom Cypress command:

```javascript
Cypress.Commands.add('attachFileToInputElement', (subject) => /* .... */)
```

Reply:
Would this add it to the Cypress public API? If so I'd probably want a more generic way to handle this. 

I think the cypress-file-upload plugin does a pretty good job covering most generic use cases for attaching a file (they have the attachFile command). I saw there was an issue to support webkitdirectory, but I wanted a path attached to my file and they didn't do that. Right now they only load files from your directory and that caused issues for me.